src/js/helpers/awardSummaryHelper.js

```diff
@@ -56,3 +62,34 @@ export const getAwardTypeByRecordtypeCountyAndState = (
     // IDV or contract
     return 'nonFinancialAssistance';
 };
+
+export const datesByAwardType = (dates, awardType) => {
```

Comment:
The naming convention `xByZ` implies a map that is keyed by a certain thing and has the value of a certain thing.

so `datesByAwardType` should look like this:

```javascript
{ contract: dates, asst: dates, idv: dates }
```

Would rename this map to conform to this convention or just use an array or possibly something like: `datesByDateType`